annotations/views/rest_views.py

```diff
@@ -493,8 +501,71 @@ def _relabel(datum):
                 'identifier': 'uri'
             }
 
-            return {_fields.get(k, k): v for k, v in datum.iteritems()}
-        return Response({'results': map(_relabel, [c.data for c in concepts])})
+            return {_fields.get(k, k): v for k, v in datum.iteritems() }
+        results = map(_relabel, [c.data for c in concepts])
+        
+        for result in results:
+            concepts = []
+            if result["identities"]: # if identities exist append the first identitiy to the list so that we can filter out other identities against it
+                concepts = result["identities"][0]["concepts"]
```

Comment:
you don't need this if, do you? if you start with an empty set, then first concepts list - [] will give you everything in the concept list.

Reply:
What I mean is that you can loose the whole if block and start with an empty concept list. the difference between an empty list and the first iteration should be everything that's in ident.